samcli/lib/package/s3_uploader.py

```diff
@@ -101,6 +101,7 @@ def upload(self, file_name, remote_path):
                     file_name, self.bucket_name, remote_path, additional_args, [print_progress_callback]
                 )
             else:
+                sys.stderr.flush()
```

Comment:
Is flush() required here? Can you try running the failing test without flush()?

Reply:
It is not required, will remove this line